packages/web-components/src/components/ic-checkbox/ic-checkbox.tsx

```diff
@@ -109,9 +109,11 @@ export class Checkbox {
     }
   }
 
+  private icTextFieldString: string = "ic-text-field";
```

Comment:
Take the uppercase into consideration here: https://javascript.info/variables#uppercase-constants
Probably won't need to add 'String' at end as well.